src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -276,20 +281,27 @@ public void update() {
             }
 
 
-            if (fuelRemaining == 0 && machineActive && machineTier == machineTier.TIER_0) {
+
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Equality tests should not be made with floating point values. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1244)